include/zen-remote/server/peer.h

```diff
@@ -8,6 +8,7 @@ struct IPeer {
   virtual ~IPeer() = default;
 
   virtual std::string host() = 0;
+  virtual bool wired() = 0;
```

Comment:
@liwii @watasuke102 You can use this to detect whether the peer is wired peer or not.

Reply:
𝗡𝗶𝗰𝗲 𝘄𝗼𝗿𝗸